### PR TITLE
Jack Gay: Feature/jack databasestructure

### PR DESCRIPTION
**Pull request by j4ckp0w15g4y**
I have completed the database structure, and relationships between the User, Material, and Location tables

### CODE REVIEW

File: server/db/models.js [hunk 1]
```diff
@@ -43,7 +43,19 @@ const User = db.define('user', {
   user.password = hashedPassword
 });
 
+/* I think these relationships make sense, but of course feel free to counter! The onDelete is in place so that if a user deletes a location from their 'favorites' list*/
+
 User.hasMany(Material)
+Material.belongsTo(Material)
```
Comment (anpato) [inline server/db/models.js]: Question about this line here, why are we telling material that it belongs to itself?

File: server/db/models.js [hunk 2]
```diff
@@ -18,6 +18,35 @@ const Location = db.define('location', {
...
+})
+
+const User = db.define('user', {
+    name: Sequelize.STRING,
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+      validate: {
+        isEmail: true
```
Comment (anpato) [inline server/db/models.js]: Can we also add username as an option? Some people prefer logging in with usernames over emails, but we should all agree on this.

File: server/db/models.js (hunk 1 above)
Comment (j4ckp0w15g4y) [reply server/db/models.js]: Sorry this was a typo, it was meant to be 'Material.belongsTo(User)'

Comment (anpato) [reply server/db/models.js]: Gotcha, let's get that squared away and then we can discuss usernames tomorrow and it'll be ready to merge

**Comment by anpato** [comment]
LGTM

**Pull request merged**